app/javascript/utilities/http/csrfToken.js

```diff
@@ -0,0 +1,30 @@
+const MAX_RETRIES = 30;
+const RETRY_INTERVAL = 250;
+
+export function getCSRFToken() {
```

Comment:
Created a `getCSRFToken` function that forms part of the utilities and can be imported. This will eventually replace all instances of `getCsrfToken`. I named them slightly different so we could distinguish between them.